purchases/src/main/kotlin/com/revenuecat/purchases/customercenter/CustomerCenterConfigData.kt

```diff
@@ -336,6 +339,7 @@ data class CustomerCenterConfigData(
                     UNKNOWN_STORE -> "Unknown"
                     CARD_STORE_PROMOTIONAL -> "Via Support"
                     RESUBSCRIBE -> "Resubscribe"
+                    SUBSCRIBE -> "Subscribe"
```

Comment:
I am thinking now... what if the offering only has one time purchases? This button will say subscribe, but there won't be subscriptions

Reply:
The copy is editable now, but I agree that we can make it more visible from the dashboard